bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableIO.java

```diff
@@ -704,6 +738,35 @@ public boolean advance() throws IOException {
       }
     }
 
+    private void resetScanner() throws IOException {
+      CloudBigtableScanConfiguration scanConfiguration = source.getConfiguration();
+      Scan scan;
+      if (lastScannedRow != null) {
+        scan =
+            scanConfiguration
+                .toBuilder()
+                .withKeys(lastScannedRow.getRow(), scanConfiguration.getStopRow())
```

Comment:
can you append a zero byte to the start key so that it start right after the last seen row key?